src/sections/Join/index.js

```diff
@@ -1,9 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 
+import { FormButton } from 'Elements';
+import {
+  joinLayout, formBlock,
+  formTitle, formSubtitle, formButton,
+  joinForm, inputFields, cta, ctaText, ctaLogo
+} from './style.css';
+
+import GitterLogo from 'Elements/GitterLogo';
 
 const Join = () => {
-  return(
-    <div>JOIN</div>
+  const [email, setEmail] = useState('');
+  
+  function handleSubmit(event) {
+    alert('form submitted');
+    event.preventDefault();
+  }
+
+  return (
+    <section id="join" className={joinLayout}>
+      <div className={formBlock}>
+        <h1 className={formTitle}>Join</h1>
+        <span className={formSubtitle}>Subscribe to monthly meetup events & get notified of upcoming events like community workshops, hackathons, presentations and recordings.</span>
+
+        <form id="joinForm" className={joinForm} onSubmit={handleSubmit}>
+          <fieldset className={inputFields}>
```

Comment:
we can get rid of the single fieldset here. only one element to group doesn't make much sense

Reply:
also, `inputFields` doesn't appear to be used